src/enums/TodoStatusOptions.ts

```diff
@@ -0,0 +1,5 @@
+export enum TodoStatusOptions {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
```

Comment:
Yep, I think it's a good case for using enum)

Reply:
@Mykola-Maik thanks)